sdk/src/azure/core/az_mqtt5_rpc_client_codec.c

```diff
@@ -1,83 +1,420 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#include <azure/core/az_mqtt5_rpc_client.h>
+#include <azure/core/az_mqtt5_rpc.h>
+#include <azure/core/az_mqtt5_rpc_client_codec.h>
 #include <azure/core/az_result.h>
 #include <azure/core/internal/az_log_internal.h>
+#include <azure/core/internal/az_precondition_internal.h>
+#include <azure/core/internal/az_result_internal.h>
+#include <azure/iot/az_iot_common.h>
```

Comment:
I don't think this should be included here

Reply:
We need it for the `AZ_ERROR_IOT_TOPIC_NO_MATCH` error